### PR TITLE
feat: Auth

### PR DESCRIPTION
**Pull request by Shivani-1524**
Added Auth and Form Validation 

Files to review:
AuthPages Folder
TemplatePage.js
authSlice.js
Navbar.js

### CODE REVIEW

**Review by prankurpandeyy — APPROVED** [review]

**Comment by prankurpandeyy** [comment]
UI and color are good but I think if you can give some opacity to background would look more attractive

File: src/App.css [hunk 1]
```diff
@@ -1,51 +1,162 @@
...
 }
 
-.links a {
-  color: #d9d9d9;
-  font-size: 1.2rem;
-  margin: 1rem;
+footer .icon-btn{
+  width: 2.3rem;
+  color: var(--primary-color);
+  padding: 10px;
```
Comment (dipen2000) [inline src/App.css:147]: Use rems or ems here

File: src/Components/CreatePost/createpost.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.post-input{
+    background-color: var(--dark-bg);
+    margin: auto;
+    display: block;
+    width: 100%;
+    min-height: 10rem;
+    border: none;
+    padding: 20px;
```
Comment (dipen2000) [inline src/Components/CreatePost/createpost.css:8]: here too

File: src/Components/CreatePost/createpost.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    color: var(--white);
+}
+.post-media-row.flex-row{
+    justify-content: space-between;
+    padding: 10px 30px;
+    padding-left: 30px;
```
Comment (dipen2000) [inline src/Components/CreatePost/createpost.css:14]: same here

**Review by dipen2000 — APPROVED** [review]
The code is readable and the UI looks good , you have used px units for sizing at some place s you gotta change it to rems or ems other than that it looks good to go.

One more thing Have the name of your app in the title in the browser tab.

**Comment by Shivani-1524** [comment]
> The code is readable and the UI looks good , you have used px units for sizing at some place s you gotta change it to rems or ems other than that it looks good to go.
> 
> One more thing Have the name of your app in the title in the browser tab.

i will give the title and the favicon at the end, thanks for the compliment

**Pull request merged**